lightspeed/modules/proc_obtain-config-wca-and-model-id.adoc

```diff
@@ -25,7 +27,7 @@ For information about how to obtain a WCA API key and model ID from {ibmwatsonxc
 .. Optional: Click *Test model ID* to validate the model ID.
 .. Click *Save*.
 +
-When the WCA API key and model ID is successfully validated, {LightspeedShortName} is connected to your {ibmwatsonxcodeassistant} instance.
+When the API key and model ID is successfully validated, {LightspeedShortName} is connected to your {ibmwatsonxcodeassistant} instance.
 
 [role="_additional-resources"]
 == Additional resources
```

Comment:
```suggestion
.Additional resources
```

Reply:
I know it's not in the PR, but adding this suggestion so that "additional resources" doesn't appear in the table of contents

Reply:
This is done, thank you for catching it.